config/routes.rb

```diff
@@ -3,7 +3,8 @@
   namespace :api do
     namespace :v1 do
       resources :events, only: [:index, :show]
-      resources :users, only: [:show] 
+      resources :users, only: [:show, :update]
+      resources :users, only: [:show]
```

Comment:
This is weird - must have come in when you rebased. Per discussion, best for you to remove this line locally and run test suite to make sure everything is still fine.